erdpy_network/proxy_network_provider_test.py

```diff
@@ -106,3 +108,63 @@ def test_get_sc_invoking_tx(self):
         assert result.is_completed == True
         assert len(result.contract_results.items) > 0
         assert result.data == 'issue@54455354@54455354@03e8@00@63616e4d696e74@74727565@63616e4275726e@74727565@63616e4368616e67654f776e6572@74727565@63616e55706772616465@74727565'
+
+    def test_send_transactions(self):
+        first_tx = FirstTransaction()
+        second_tx = SecondTransaction()
+        third_tx = ThirdTransaction()
+
+        transactions = [first_tx, second_tx, third_tx]
+
+        expected_hashes = [
+            "6e2fa63ea02937f00d7549f3e4eb9af241e4ac13027aa65a5300816163626c01",
+            "37d7e84313a5baea2a61c6ab10bb29b52bc54f7ac9e3918a9faeb1e08f42081c"
+        ]
+
+        assert self.proxy.send_transaction(transactions[0]) == expected_hashes[0]
```

Comment:
Now I see that perhaps we should replicate erdjs' logic for the returned response, as well?

https://github.com/ElrondNetwork/elrond-sdk-erdjs-network-providers/blob/main/src/proxyNetworkProvider.ts#L119

That is, returning a list that contains the hashes for accepted transactions / null items for rejected ones. 

Can be done in another PR, though.

Reply:
I'll put it on the list.